near-token-factory/src/lib.rs

```diff
@@ -42,39 +54,60 @@ pub struct Contract {
 }
 
 // TODO: Add pausable
-// TODO: Add access control
 #[near_bindgen]
 impl Contract {
     /// Initializes the contract. The locker account id MUST be the NEAR
     /// representative of the Aurora address of the locker contract created
     /// using the Cross Contract Call interface.
     #[init]
-    pub fn new(aurora: AccountId, locker: aurora_sdk::Address) -> Self {
+    pub fn new(aurora: AccountId, locker: aurora_sdk::Address, admin: Option<AccountId>) -> Self {
```

Comment:
```suggestion
    pub fn new(aurora: AccountId, locker: aurora_sdk::Address, super_admin: Option<AccountId>) -> Self {
```

See similar suggestion above for the motivation.

Reply:
Done in ccd84c46af2278ef9b81fb21dcfa824d28ea41c6